Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:5.12
```

Comment:
So, according to Node versioning documentation, we shouldn't be using the 5.x series.

https://github.com/nodejs/LTS#lts_schedule

Looks like 6.x is going to be the next LTS very soon, so I think we should try switching to that.

Given our test coverage I'd be comfortable pining this as loosely as this `FROM node:6`